nodes/mtdevice.py

```diff
@@ -1699,6 +1811,46 @@ def get_settings(arg):
     settings = timestamp | orient_mode | calib_mode | NED
     return settings
 
+def get_synchronization_settings(arg):
+    """Parse command line synchronization-settings argument."""
+    # Parse each field from the argument
+    settings = arg.split(',')
+    settings = tuple([int(i) for i in settings]) 
+    # Check settings
+    valid = False
+    if settings[0] in (3, 4, 8, 9, 11) and \
+       settings[1] in (0, 1, 2, 4, 5, 6) and \
+       settings[2] in (1, 2, 3) and \
+       settings[3] in (0, 1):
+        valid = True
+    else:
+        valid = False
+
+    if valid:
```

Comment:
I can see the interest of having a Boolean sometimes but here, I'd prefer if you could merge the two tests.